Updates to upgrade procedures (#65388)
openshift/openshift-docs: modules/install-acs-operator-cloud.adoc

+
If you select automatic updates, when a new version of the Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator.
+

> If you select manual updates, when a newer version of the Operator is available, OLM creates an update request. As a cluster administrator, you must manually approve the update request to update the Operator to the latest version. Red{&nbsp}Hat recommends maintaining secured cluster components at the same version as Central it is connected to for optimal compatibility. For {product-title-managed-short}, the software version for Central is automatically updated. See the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix] for more information.

Reviewer: `{&nbsp}` does not render correctly.

![image](https://github.com/openshift/openshift-docs/assets/537715/c7e85c77-900d-46cf-afd8-ba4356b27d00)

----

I realized this page is dedicated to ACS CS therefore the only case to install ACS Operator is for Secured Cluster. There is no need to install ACS Central in ACS CS _by definition_.

Therefore, we should reword this paragraph with Secured Cluster being the only thing in the context. For Secured Cluster connected to ACS CS, we (or Red Hat) recommend automatic upgrades.

```suggestion
If you select manual updates, when a newer version of the Operator is available, OLM creates an update request. As a cluster administrator, you must manually approve the update request to update the Operator to the latest version.

Red{&nbsp}Hat recommends enabling automatic upgrades for Operator in {product-title-managed-short}. See the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix] for more information.
```

If you select automatic updates, when a new version of the Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator.
+
If you select manual updates, when a newer version of the Operator is available, OLM creates an update request. As a cluster administrator, you must manually approve the update request to update the Operator to the latest version.
+

> Red{&nbsp}Hat recommends enabling automatic upgrades for Operator in {product-title-managed-short}. See the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix] for more information.

Author: Argh, thanks for catching this.
